package.json

```diff
@@ -69,5 +50,26 @@
     "webpack": "^1.13.1",
     "webpack-dev-middleware": "^1.6.1",
     "webpack-hot-middleware": "^2.10.0"
+  },
+  "dependencies": {
+    "express": "^4.13.4",
```

Comment:
Is express really a dependency (as opposed to `devDependency`?). We're only using it for development right?